strings_lab_assignment.js

```diff
@@ -0,0 +1,78 @@
+// 1. console.log the first character of a string.
+console.log("----#1------");
+let str = "karen"
+console.log(str[0]);
+
+// 2. console.log the length of a string.
+console.log("----#2------");
+console.log(str.length);
+
+// //3. console.log the last character of any string.
+console.log("----#3------")
+console.log(str[str.length-1]);
+
+//4. Create a code block that takes a single string variable, and
+//returns a copy of the string with the last letter capitalized.
+//For example:
+
+// ```js
+// 'jimmy'
+// // => 'jimmY'
+// ```
+console.log("----#4------");
+let lastCap = str[str.length-1].toUpperCase();
+// console.log(lastCap);
+console.log(str.slice(0,4) + lastCap);
```

Comment:
What would happen if I change str to "Alejandro"? Would this still give us AlejandrO?